rosidl_generator_py/resource/_msg.py.em

```diff
@@ -140,7 +140,7 @@ for member in message.structure.members:
 @[end for]@
 @[for member in message.structure.members]@
 @[  if member.has_annotation('default')]@
-            '@(member.name.upper())__DEFAULT': @(value_to_py(member.type, member.get_annotation_value('default')['value'])),
+            '@(member.name.upper())__DEFAULT': @(value_to_py(member.type, member.get_annotation_value('default')['value'])),  # noqa: Q003
```

Comment:
Instead of applying the warning suppression to any kind of content I would suggest to update the `primitive_value_to_py()` function instead to use the proper quoting.

Same below.

Reply:
The problem is figuring out the proper quoting.  To do that, we actually have to walk the entire string and see whether there are any embedded single or double quotes, and then use the opposite.  We'll also need to handle escaped quotes properly.  In some cases, we won't be able to figure out one, so we'll still have to arbitrarily choose one.

It just doesn't seem worth it either for development time or for runtime.  What does the check really buy us?

Reply:
That the generated code follows our code style guidelines.

Reply:
But that's just not a good enough reason, particularly for generated code, in my opinion.  All of this stuff has a cost in terms of time and complexity, and doing busywork like this just doesn't seem to be a good use of either.

Reply:
I simply implemented the logic in a couple of minutes: see #43